near-sdk/src/lib.rs

```diff
@@ -9,6 +9,9 @@ pub use near_sdk_macros::{
     ext_contract, metadata, near_bindgen, BorshStorageKey, FunctionError, PanicOnDefault,
 };
 
+#[cfg(feature = "unstable")]
+pub use near_sdk_macros::near_abi;
```

Comment:
@austinabell am I correct in saying that this won't stop `near_abi` from being compiled without unstable feature? Is there a way I can stop that without introducing unstable feature in near-sdk-macros crate?

Reply:
This looks right to me. Rust won't make use of `near_abi` unless it's referenced and used somewhere. An export will just allow users to reference it, meanwhile, the feature flag will only enable the export when the unstable flag is set.

If you want to add the unstable flag to near-sdk-macros as well, you can nest feature flags, such that when near-sdk unstable is set, near-sdk-macro's unstable flag is also set like so:
```toml
# near-sdk
[features]
unstable = ["near-sdk-macros/unstable"]

# near-sdk-macros
[features]
unstable = []
```

Reply:
yeah, what Phuong mentions here with having an unstable feature flag for macros is probably required. Having the dead codegen is probably not something we want for those not using this

Reply:
Gotcha, should be better now